ui/current_1_scalable.qml

```diff
@@ -23,46 +23,163 @@ device not conforming to the Mark II screen's form factor.
+        height: parent.height
+        color: "black"
 
-    LottieAnimation {
-        // Animation depicting the current weather conditions (e.g. sunny, cloudy, etc.)
-        id: weatherAnimation
-        Layout.alignment: Qt.AlignHCenter
-        Layout.preferredWidth: Math.min(root.contentWidth, proportionalGridUnit * 50)
-        Layout.preferredHeight: Layout.preferredWidth
+        WeatherLocation {
+            id: weatherLocation
+            fontSize: parent.height > parent.width ? parent.width * 0.10 : 45
+        }
 
-        source: Qt.resolvedUrl(getWeatherImagery(sessionData.weatherCode))
+        GridLayout {
+            id: weather
+            anchors.left: parent.left
+            anchors.right: parent.right
+            anchors.bottom: parent.bottom
+            anchors.top: weatherLocation.bottom
+            anchors.topMargin: Mycroft.Units.gridUnit * 2
+            columns: 2
+            columnSpacing: Mycroft.Units.gridUnit * 2
 
-        loops: Animation.Infinite
-        fillMode: Image.PreserveAspectFit
-        running: true
+            Item {
+                //First column in grid
+                id: currentConditions
+                Layout.fillWidth: true
+                Layout.fillHeight: true
 
-        // Debug:
-        onSourceChanged: {
-            console.log(getWeatherImagery(sessionData.weatherCode));
-        }
-        onStatusChanged: {
-            console.log(weatherAnimation.status, errorString);
-        }
-    }
+                Item {
+                    width: parent.width
+                    height: parent.height / 2
+
+                    Image {
+                        //Image depicting the current weather conditions (e.g. sunny, cloudy, etc.)
+                        id: weatherAnimation
+                        anchors.bottom: parent.bottom
+                        anchors.bottomMargin: Mycroft.Units.gridUnit
+                        anchors.horizontalCenter: parent.horizontalCenter
+                        width: parent.width - Mycroft.Units.gridUnit * 2
+                        height: parent.height > parent.width ? width : parent.height - Mycroft.Units.gridUnit * 2
+                        source: Qt.resolvedUrl(getWeatherImagery(sessionData.weatherCode))
+                        fillMode: Image.PreserveAspectFit
+                    }
+                }
+
+                Label {
+                    //Current temperature in the configured temperature unit
+                    id: temperature
+                    font.weight: Font.Bold
+                    font.pixelSize: parent.height > parent.width ? parent.width * 0.65 : parent.height * 0.65
+                    anchors.baseline: parent.bottom
+                    anchors.baselineOffset: -Mycroft.Units.gridUnit
+                    horizontalAlignment: Text.AlignHCenter
+                    width: parent.width
+                    rightPadding: -font.pixelSize * 0.1
+                    font.family: "Noto Sans Display"
```

Comment:
This is probably left in from before, but we've been trying to move away from setting the font-family too much to allow projects to use their own preferred fonts. 

If we need it to ensure sizing etc, we've been moving to "Noto Sans" without the "Display"

Reply:
Has been fixed, I have set the font.family to general "Noto Sans"